### PR TITLE
MM-T4276 :  E2E for set up Board emoji

### PR DESCRIPTION
**Pull request by adj2908**
#### Summary

This PR adds a cypress e2e test for setting up board emoji/Icon.

#### Ticket Link

The ticket is from the focalboard repo.
[GitHub issue link
](https://github.com/mattermost/focalboard/issues/1814)

#### Release Note

```
NONE
```

### CODE REVIEW

**Review by furqanmlk — APPROVED** [review]
Thanks @adj2908  LGTM. [e2e](https://automation-dashboard.vercel.app/cycle/324757) passed

**Review by yasserfaraazkhan — APPROVED** [review]
🎉

**Comment by adj2908** [comment]
@yasserfaraazkhan @furqanmlk Thanks for the approval. Can you guys please merge the PR as I do not have the permissions to merge this. Thanks.

**Comment by adj2908** [comment]
@yasserfaraazkhan  @furqanmlk @saturninoabril  Can you guys please merge the PR as I do not have the permissions to merge this. Thanks.

**Comment by saturninoabril** [comment]
Thanks @adj2908 for your contribution! I'll run the E2E test first and see how it goes -- will run once we fix the E2E pipeline which is having an issue right now. Then this is good to merge.

File: e2e-tests/cypress/tests/integration/boards/create_board_spec.ts [hunk 1]
```diff
@@ -98,6 +98,72 @@ describe('Create and delete board / card', () => {
         cy.findByText('for testing purposes only').should('be.visible');
     });
 
+    it('MM-T4275 Set up Board emoji', () => {
```
Comment (saturninoabril) [inline e2e-tests/cypress/tests/integration/boards/create_board_spec.ts:101]:
```suggestion
    it('MM-T4276 Set up Board emoji', () => {
```

**Comment by saturninoabril** [comment]
Test passed in CI. This is now good to merge.

Note: I've updated the test case ID - https://github.com/mattermost/mattermost-server/pull/22887#discussion_r1170706018.

**Pull request merged**